README.md

```diff
@@ -52,6 +52,16 @@ mvn compile exec:java \
   -Dorg.apache.commons.logging.simplelog.log.org.apache.http.wire=INFO
 ```
 
+A sample that uses dynamic proxy to create the `OpenSearchClient` so you can write cross-cutting concerns once is also available in [src/main/java/LoggingClientExample.java](src/main/java/LoggingClientExample.java).
```

Comment:
Explain what those cross cutting concerns are. 

A sample that uses a dynamic proxy to intercept calls and do X Y Z?

Reply:
Updated the readme and rebased, please see if it reads better now?